saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -308,3 +312,245 @@ def perform_mutation(cls, _root, info, **data):
         fulfillment.refresh_from_db(fields=["status"])
         order.refresh_from_db(fields=["status"])
         return FulfillmentCancel(fulfillment=fulfillment, order=order)
+
+
+class OrderRefundLineInput(graphene.InputObjectType):
+    order_line_id = graphene.ID(
+        description="The ID of the order line to refund.", name="orderLineId"
+    )
+    quantity = graphene.Int(
+        description="The number of items to be refunded.", required=True,
+    )
+
+
+class OrderRefundFulfillmentLineInput(graphene.InputObjectType):
+    fulfillment_line_id = graphene.ID(
```

Comment:
IMO there is no need to add id suffix. In our code base we are using just name eg `fulfillment_line`. We should be consistent according to our codebase.